monitor.py

```diff
@@ -971,13 +997,38 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                 #     • SuitName
                 #     • LoadoutID
                 #     • Loadoutname
-                pass
+                # alpha4:
+                # { "timestamp":"2021-04-29T10:35:55Z", "event":"RenameSuitLoadout", "SuitID":1698365752966423,
+                # "SuitName":"explorationsuit_class1", "SuitName_Localised":"Artemis Suit", "LoadoutID":4293000003,
+                # "LoadoutName":"Art L/K" }
+                loadout_id = self.suit_loadout_id_from_loadoutid(entry['LoadoutID'])
+                try:
+                    self.state['SuitLoadouts'][loadout_id]['name'] = entry['LoadoutName']
+
+                except KeyError:
+                    logger.exception(f"loadout slot id {loadout_id} doesn't exist, not in last CAPI pull ?")
 
-            # `BuySuit` has no useful info as of 4.0.0.13
             elif event_type == 'BuySuit':
-                # alpha4 - should have 'SuitID'
+                # alpha4 :
+                # { "timestamp":"2021-04-29T09:03:37Z", "event":"BuySuit", "Name":"UtilitySuit_Class1",
+                # "Name_Localised":"Maverick Suit", "Price":150000, "SuitID":1698364934364699 }
+                new_suit = {
+                    'name':      entry['Name'],
+                    'locName':   entry.get('Name_Localised', entry['Name']),
+                    'id': None,  # Is this an FDev ID for suit type ?
+                    'suitId':    entry['SuitID'],
+                    'slots':     [],
+                }
+                if self.state['Suits'] is None:
+                    self.state['Suits'] = {entry['SuitID']: new_suit}
+
+                else:
+                    self.state['Suits'].update(
+                        {entry['SuitID']: new_suit}
```

Comment:
Does this really end up doing anything different here?
If suits is None: Add the suit under the ID
otherwise: Add the suit under the ID

All that changes is that update is used where on both it could be if suits is None make it an empty dict, then regardless set the key for this suits id

Reply:
Yeah, I'll change this.  It's a product of the direction I came at it, and which bit got written first.